object-cache.php

```diff
@@ -508,64 +540,161 @@ function switch_to_blog( $blog_id ) {
 		$this->blog_prefix = ( is_multisite() ? $blog_id : $table_prefix );
 	}
 
-	function colorize_debug_line( $line ) {
+	function colorize_debug_line( $line, $trailing_html = '' ) {
 		$colors = array(
 			'get' => 'green',
-			'get_multi' => 'green',
+			'get_local' => 'lightgreen',
+			'get_multi' => 'fuchsia',
 			'set' => 'purple',
+			'set_local' => 'orchid',
 			'add' => 'blue',
 			'delete' => 'red',
+			'delete_local' => 'tomato',
+			'slow-ops' => 'crimson',
 		);
 
 		$cmd = substr( $line, 0, strpos( $line, ' ' ) );
 
 		// Start off with a neutral default color...
 		$color_for_cmd = 'brown';
-
 		// And if the cmd has a specific color, use that instead
 		if ( isset( $colors[ $cmd ] ) ) {
 			$color_for_cmd = $colors[ $cmd ];
 		}
 
-		$cmd2 = '<span style="color:' . esc_attr( $color_for_cmd ) . '">' . esc_html( $cmd ) . '</span>';
+		$cmd2 = "<span style='color:" . esc_attr( $color_for_cmd ) . "; font-weight: bold;'>" . esc_html( $cmd ) . "</span>";
 
-		return $cmd2 . esc_html( substr( $line, strlen( $cmd ) ) ) . "\n";
+		return $cmd2 . esc_html( substr( $line, strlen( $cmd ) ) ) . "$trailing_html\n";
 	}
 
-	function stats() {
-		if ( $this->stats_callback && is_callable( $this->stats_callback ) ) {
-			return call_user_func( $this->stats_callback );
+	function js_toggle() {
+		echo "
+		<script>
+		function memcachedToggleVisibility( id, hidePrefix ) {
+			var element = document.getElementById( id );
+			if ( ! element ) {
+				return;
+			}
+
+			// Hide all element with `hidePrefix` if given. Used to display only one element at a time.
+			if ( hidePrefix ) {
+				var groupStats = document.querySelectorAll( '[id^=\"' + hidePrefix + '\"]' );
+				groupStats.forEach(
+					function ( element ) {
+					    element.style.display = 'none';
+					}
+				);
+			}
+
+			// Toggle the one we clicked.
+			if ( 'none' === element.style.display ) {
+				element.style.display = 'block';
+			} else {
+				element.style.display = 'none';
+			}
 		}
+		</script>
+		";
+	}
+
+	function stats() {
+		$this->js_toggle();
 
-		echo "<p>\n";
+		echo '<h2><span>Total memcache query time:</span>' . number_format( sprintf( '%0.1f', $this->time_total * 1000 ), 1, '.', ',' ) . 'ms</h2>';
+		echo "\n";
+		echo '<h2><span>Total memcache size:</span>' . esc_html( size_format( $this->size_total, 2 ) ) . '</h2>';
```

Comment:
[Moved](https://github.com/Automattic/wp-memcached/pull/53/commits/f77ea45c107323ef3e397e7f31fe3caaac6710ff) to using WPs `size_format` instead of a custom implementation.